config/locales/en.yml

```diff
@@ -49,20 +49,22 @@ en:
     pgde_with_qts: "PGDE with QTS"
   location_filter:
     errors:
-      no_option: "Please choose an option"
-      unknown_location: "We couldn’t find this location, please check your input and try again"
-      missing_location: "Please enter a postcode, city or town in England"
-      missing_provider: "We couldn’t find this provider, please check your information and try again"
-      blank_provider: "You need to add some information"
-      invalid_provider: "Please enter a minimum of two characters"
+      no_option: Select an option to find courses
+      unknown_location: Enter a real city, town or postcode
+      missing_location: Enter a city, town or postcode
+      missing_provider: Enter a real school, university or training provider
+      blank_provider: Enter a school, university or other training provider
     fields:
       location: "Postcode, town or city"
       provider: "Training provider"
   subject_filter:
     errors:
-      no_option: "Please choose at least one subject"
+      no_option: Select at least one subject
+  qualification_filter:
+    errors:
+      no_option: Select at least one qualification
   cookie_preferences:
-    no_option_error: "Please choose an option"
+    no_option_error: Select yes if you accept Google Analytics cookies
```

Comment:
@gregknight1 spotted another one here. Let me know if this is ok, or if you have any better suggestions?

Reply:
@frankieroberto That's fine - thanks.